config/samples/toolchain.dev.openshift.com_v1alpha1_socialevent.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: toolchain.dev.openshift.com/v1alpha1
```

Comment:
this also came from operator-sdk, but I'm not sure we need it

Reply:
You can drop it. 
Technically speaking - we should have the sample for each of our CRD, but let's be practical here - the samples are not supposed to be directly used by anyone, really. As soon as we have to add them (because of any reason) then we can add them, but for now, let's make our lives and the maintenance easier :-)

Reply:
yeah, that was my feeling, too ;)

Reply:
removed in 8a32d92

Reply:
removed in https://github.com/codeready-toolchain/host-operator/pull/624/commits/8a32d924810cca8098cff6c76cf003c9bf2153db